### PR TITLE
Features

### PR DESCRIPTION
**Pull request by SigmaSam**
Build a working library with JavaScript and Tailwind:

- Allow the user to add new books.
- Allow the user to deleted books.
- Added the option to toggle between if a book added by the user was read or not.

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### INVALID REQUEST ♻️ 🛑 
Hello, @SigmaSam  and @julian3493. This is a great effort to submit your project for review. However, for your project to be reviewed, it must meet the basic minimum requirement. please read again about those requirements [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/). 

I am marking your project as invalid because of the following ⬇️ 

- There must be a descriptive readme file in your project. Kindly I think you forgot to include this. Kindly add a descriptive readme file and submit your project for another review ⬇️ 

descriptive_readme (1)

### if you have any question, kindly reach me on slack @peterrobert


### HAPPY CODING 💻

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello @SigmaSam and @julian3493.

**I can see a lot of effort in this project and you did a good job** 👏 👏 👏 

## The highlights: 🔆 
1. `Git-flow` is correct. 💯 
2.  `Linters` is well set-up. 💯 
3.  Descriptive `README` file. 👍 
4. Very nice PR message and commit messages. 💯 
5. Using `Tailwind` is a bonus. ⭐ ⭐ 

 ## What to do next:  
**Some small works need to be done and then you are finished with this project.**  🛠️ 

**Remember you need to make the required changes.** 👇 👇 

**[REQUIRED]** ⏩
- [x] There is a typo in your Book Library which is "Readed?". It should be `READ`.
- [x]  When you are importing js file in the HTML, using `type=" module"` will prevent opening the page in the browser, So, you need to add instructions in `README` to Open with Live server. Or, you can simply remove that piece of code.
- [x] In your `main.js` there is an inline commented code in 26. Please remove it.
- [x]  It is not a good practice to disable eslint in the middle of code, try to solve the error.
- [x]  Remove the inline comments from the stylesheets of styles folder, But not the 
**comments about code** `/* Navigation */`
##

**[Optional]** ⏩
- [ ] Using ES6 functionalities would be suggested.
- [ ]  Consider adding some pre-added books in your library, so that the viewer can see how the library is looking.
##

**Please make the changes and ask for a re-review.**

**Don't worry about the change requirements, every time with some flows you are learning something new.** ☺️ ☺️ 

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. Happy Coding 💻

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+<a href="https://rawcdn.githack.com/SigmaSam/Library-JS/adf13634f6767c39fbe06ca410084dc48902ddbc/index.html" target="_blank">Live version</a>
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+- Clone this repository.
```
Comment (whiz25) [inline README.md]: - What's the command for cloning?
- After cloning then what? Add command for checking directory and checking out the `features` branch
- How do I open the app? Be explicit in your instructions. Don't assume the user knows how.

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
+
+// Book Class
```
Comment (whiz25) [inline scripts/main.js]: - Remove all comments from your code. It makes the code look ugly.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work @SigmaSam . You did well making the requested changes. 
Let's fix a few things that will make this project more professional before merging.
- [ ] Remove comments from your codebase. They make your code look ugly
- [x] Your README instructions are not explicit enough. Check my comments for suggestions


### Martin Afani

**Review by wathigo — APPROVED** [review]
## APPROVED :heavy_check_mark:  
Great job on this milestone guys.
All the requested changes have been met.

**You may proceed to the next milestone.**

**Happy Coding :computer: 
Simon Wathigo**

**Pull request merged**